package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "replicated-lint",
   "author": "Replicated, Inc.",
-  "version": "0.14.9",
+  "version": "0.14.10",
```

This is going to need to be bumped...
```suggestion
  "version": "0.14.11",
```